tests/module_api/test_api.py

```diff
@@ -762,7 +762,7 @@ def test_send_local_online_presence_to_workers(self):
 
 
 def _test_sending_local_online_presence_to_local_user(
-    test_case: HomeserverTestCase, test_with_workers: bool = False
+    test_case: BaseMultiWorkerStreamTestCase, test_with_workers: bool = False
```

Comment:
I don't think this is correct? Since we call this function in `ModuleApiTestCase.test_send_local_online_presence_to` and `ModuleApiTestCase` is a `HomeserverTestCase`, not a `BaseMultiWorkerStreamTestCase`.

Reply:
Hmm. `test_case` must be a `BaseMultiWorkerStreamTestCase`, but only if `test_with_workers` is True` (as only `BaseMultiWorkerStreamTestCase` has a `make_worker_hs` method).

So it works in the case of `ModuleApiTestCase.test_send_local_online_presence_to`, as it sets `test_with_workers` to False.

I've added an assert for the correct type and updated the docstring in 4f5655704890d631e0543a309b37b1dccbbb39d2. Does that approach seem reasonable?

Reply:
Yep, sounds good to me.